torch_lr_finder/lr_finder.py

```diff
@@ -286,7 +286,7 @@ def _validate(self, dataloader):
         running_loss = 0
         self.model.eval()
         with torch.no_grad():
-            for inputs, labels in dataloader:
+            for inputs, labels, *_ in dataloader:
```

Comment:
This is not needed since the `__next__` method only returns the pair `(inputs, labels)`

Reply:
But would it be safer to keep it in case someone use a dataloader with custom `DataLoaderIter`?

Like the implementation shown in the link below, `self._next_data()` will be invoked in `__next__`, but the actual content of `data` returned by `self._next_data()` is determined by user's implementation. (I have done something similar before 😆)
https://github.com/pytorch/pytorch/blob/7f73f1d/torch/utils/data/dataloader.py#L344

Reply:
Ya, you are right. I was thinking that `dataloader` was wrapped in `DataLoaderIterWrapper` but it isn't; so it's perfectly possible for it to return additional values.